ui-frontend/packages/catalog-ui-search/src/main/webapp/component/fields/useTimePrefs.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { useBackbone } from '../selection-checkbox/useBackbone.hook'
+// @ts-ignore ts-migrate(7016) FIXME: Could not find a declaration file for module '../s... Remove this comment to see the full error message
+import user from '../singletons/user-instance'
+
+const useTimePrefs = () => {
+  const { listenTo } = useBackbone()
+  const [, setForceRender] = useState(Math.random())
```

Comment:
Had no idea you could do this (leave out the first param).  🤯

Reply:
☝️ Same here lol, nice stuff

Reply:
I've had to learn to keep Typescript happy 😂